ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/Utils.java

```diff
@@ -176,6 +177,12 @@ public static Boolean shouldReplicate(ReplicationSpec replicationSpec, Table tab
       return false;
     }
 
+    if (!hiveConf.getBoolVar(HiveConf.ConfVars.REPL_INCLUDE_EXTERNAL_TABLES) &&
```

Comment:
Can move this check inside below if block for replicationSpec.isInReplicationScope().

Reply:
done